packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -386,14 +444,29 @@ export interface BaseSetting {
      */
     caseSensitive?: boolean;
 
-    /** Define additional available dictionaries. */
+    /**
+     * Define additional available dictionaries.
+     *
+     * For example, you can use this to enable both English and Spanish spell-checking at the same
+     * time:
+     *
+     * ```json
+     * "language": "en-US",
+     * "dictionaryDefinitions": [
+     *   { "name": "spanish", "path": "./spanish-words.txt"},
+     * ]
+     * ```
+     */
     dictionaryDefinitions?: DictionaryDefinition[];
 
     /**
-     * Optional list of dictionaries to use.
-     * Each entry should match the name of the dictionary.
-     * To remove a dictionary from the list add `!` before the name.
-     * i.e. `!typescript` will turn off the dictionary with the name `typescript`.
+     * Optional list of dictionaries to use. Each entry should match the name of the dictionary.
+     *
+     * To remove a dictionary from the list, add `!` before the name.
+     *
+     * For example, `!typescript` will turn off the dictionary with the name `typescript`.
+     *
+     * See the [dictionaries documentation](https://cspell.org/docs/dictionaries/) for more details.
```

Comment:
Current links are: `[Dictionaries](https://cspell.org/docs/dictionaries/)` and `[Custom Dictionaries](https://cspell.org/docs/dictionaries-custom/)`

Reply:
can you offer this as a suggestion so i can click the button "commit suggestion"?